src/couch_mrview/src/couch_mrview_util.erl

```diff
@@ -525,6 +527,33 @@ validate_args(#mrst{} = State, Args0) ->
     end.
 
 
+apply_limit(ViewPartitioned, Args) ->
+    LimitType = case ViewPartitioned of
+        true -> "partition_query_limit";
+        false -> "query_limit"
+    end,
+    
+    MaxLimit = config:get_integer("query_server_config",
+        LimitType, ?MAX_VIEW_LIMIT),
+
+    % set the highest limit possible if a user has not 
+    % specified a limit
+    Args1 = case Args#mrargs.limit =:= ?MAX_VIEW_LIMIT
+        andalso MaxLimit =/= infinity of
```

Comment:
No need for this clause checking `infinity` anymore.

Reply:
I changed this line from `if Args1#mrargs.limit =:= MaxLimit -> Args1; true ->` to
`if Args1#mrargs.limit =< MaxLimit -> Args1; true ->`